docs/tutorials/basics-docker.rst

```diff
@@ -26,13 +26,20 @@ Download the Tutorial and Start KSQL
    .. code:: bash
 
        $ git clone git@github.com:confluentinc/ksql.git
+       $ cd ksql
 
-#. Navigate to the KSQL repository ``/docs/tutorials/`` directory and launch the tutorial in
+#. Switch to the correct Confluent Platform release branch:
+
+   .. code:: bash
+
+       $ git checkout 4.1.0-post
```

Comment:
Question: Should this be `4.1.0-post` (I suppose so) or `4.1.x`?  It also depends on when, as part of the release process, the `4.1.0-post` branch will be created/available.

Reply:
The `-post` branches were created to allow docs updates after the release was cut. In this case, I think the `4.1.x` branches are more appropriate.

Reply:
To double-check: It could happen though that a branch like `4.1.x` would use SNAPSHOT versions and/or "non-release" Docker image tags in `docker-compose.yml` and similar places? I don't want "release" users to end up executing in-development examples.

Reply:
Discussed offline, we agreed on `4.1.0-post`.

We leave the value hard-coded in this PR, and would do a separate PR to turn it into a variable (if needed).